emperor/util.py

```diff
@@ -532,3 +532,45 @@ def sanitize_mapping_file(data, headers):
 
     return out_lines[1::], out_lines[0]
 
+def guess_coordinates_files(dir_path):
+    """Given a directory return the file paths that can contain coordinates
+
+    Parameters
+    ----------
+    dir_path : str
+        path to the directory where coordinate files are contained
+
+    Returns
+    -------
+    list of str
+        list of filepaths pointing to the coordinates files
+
+    Notes
+    -----
+    If a path inside dir_path meets any of the following criteria, it will be
+    ignored:
+    - Is a hidden file
+    - Is named `Icon?`.
+    - Is folder
+    - Is part of the procrustes results from QIIME, see
+      transform_coordinate_matrices.py
+    """
+    coord_fps = []
+
+    for filepath in listdir(dir_path):
```

Comment:
Ahhh, now I know what you mean. I don't think we can use it, because we
need an "exclude" pattern which doesn't seem to be supported by python,
see:

```python
In [1]: from glob import glob

In [2]: glob('!(Do*)')
Out[2]: []

In [3]: ls 
Applications/
Desktop/
Documents/
Downloads/
Dropbox/
Library/
Mail/
Movies/
Music/
Pictures/
Public/
Rarefy_draft4.docx
Room andor AV Confirmation-53429.pdf
Screen Shot 2015-02-23 at 5.47.52 PM.png
Sites/
VirtualBox VMs/
git_sw/
hg_sw/
miniconda/
muttrc
svn_sw/
tmp/
workshop-schedule.pdf
```

On (Feb-23-15|18:24), Antonio Gonzalez wrote:
> > +    list of str
> > +        list of filepaths pointing to the coordinates files
> > +
> > +    Notes
> > +    -----
> > +    If a path inside dir_path meets any of the following criteria, it will be
> > +    ignored:
> > +    - Is a hidden file
> > +    - Is named `Icon?`.
> > +    - Is folder
> > +    - Is part of the procrustes results from QIIME, see
> > +      transform_coordinate_matrices.py
> > +    """
> > +    coord_fps = []
> > +
> > +    for filepath in listdir(dir_path):
> 
> Sorry for not being clear. I was suggesting glob.glob so you could retrieve specific filenames from the file system and avoid looping on the files to find if they start with . or if they have x or y in the filename. I do not believe this is necessary but I was wondering if you have considered it. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/347/files#r25223880